<launch>
    <arg name="feature_config_path" default = "$(find feature_tracker)/../config/feature_tracker/euroc_config.yaml" />
    <!-- <arg name="feature_config_path" default = "$(find feature_tracker)/../config/feature_tracker/mtuav_v3_config.yaml" /> -->

    <node name="rvizvisualisation" pkg="rviz" type="rviz" output="log" args="-d $(find plvins_estimator)/../config/vins_rviz_config.rviz" />

    <node name="pointfeature_tracker" pkg="feature_tracker" type="pointfeature_tracker_node.py" output="screen" >
        <param name="config_path" type="string" value="$(arg feature_config_path)" />
    </node>

    <node name="linefeature_tracker" pkg="feature_tracker" type="linefeature_tracker_node.py" output="screen">
        <param name="config_path" type="string" value="$(arg feature_config_path)" />
    </node>    

</launch>

